aether-odk-module/aether/odk/api/tests/files/demo-xform.xml

```diff
@@ -1,11 +1,5 @@
-<h:html
-        xmlns="http://www.w3.org/2002/xforms"
-        xmlns:ev="http://www.w3.org/2001/xml-events"
-        xmlns:h="http://www.w3.org/1999/xhtml"
-        xmlns:jr="http://openrosa.org/javarosa"
-        xmlns:odk="http://www.opendatakit.org/xforms"
-        xmlns:orx="http://openrosa.org/xforms"
-        xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+
```

Comment:
I know that this sounds weird, but I broke this file into several lines to test that our custom xml parser works. The point is to break the tags in different lines not everything in the same looooooooooong line. The previous parser crashed with this case. If you don't mind could you revert some of the breaking lines I had before :grimacing:

Reply:
No problem, for all the broken up lines or just the ones in this element?

Reply:
All of them :woman_shrugging:

Reply:
Also, is this kind of pretty-print a standard I can find? My sublime add-on doesn't do anything so nice?

Reply:
It was manual :hand: